platform/distance.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include "platform/measurement_utils.hpp"
+
+#include <string>
+
+namespace platform
+{
+class Distance
+{
+public:
+  enum class Units
+  {
+    Meters = 1,
+    Kilometers = 2,
+    Feet = 3,
+    Miles = 4,
+
+    // OSM-only
+    NauticalMiles = 5,
```

Comment:
Are they used somewhere? To convert features data in generator?

Reply:
Yes, do we really need NauticalMiles and Inches?

Reply:
I thought about using this class directly in generator (and other places) in future.
But let's leave it for the next iteration. I'll remove them now